src/Components/Web/src/Routing/NavLink.cs

```diff
@@ -74,7 +74,7 @@ protected override void OnParametersSet()
             _hrefAbsolute = href == null ? null : NavigationManager.ToAbsoluteUri(href).AbsoluteUri;
             _isActive = ShouldMatch(NavigationManager.Uri);
 
-            _class = (string?)null;
+            _class = string.Empty;
```

Comment:
Could you elaborate on why this is a good fix? `Convert.ToString` can return null so this change does not guarantee a non-null value.